flipt-client-ruby/spec/evaluation_spec.rb

```diff
@@ -90,4 +90,15 @@
       expect(resp['error_message']).to eq('invalid request: failed to get flag information default/nonexistent')
     end
   end
+
+  describe '#list_flags' do
+    it 'returns a list of flags' do
+      resp = @client.list_flags
+
+      expect(resp).to_not be_nil
+      expect(resp['status']).to eq('success')
+      expect(resp['error_message']).to be_nil
+      expect(resp['result']).to include({"enabled" => true, "key" => "flag_boolean", "type" => "BOOLEAN_FLAG_TYPE"})
```

Comment:
idk if I love using the same response object that we use for evaluation here.. wdyt?

Reply:
@markphelps What did you have in mind to change it to?

Reply:
Not sure, just trying to think from a consumers point of view. As right how This is only used for evaluation, but if we use the same structure for list flags it might be a bit confusing? I guess we could just namespace it by doing `result.flags` which would return the array of the flags?